conf/messages

```diff
@@ -385,3 +388,62 @@ removeClassOfBeneficiary.error.required = Select yes if you want to remove the c
 removeIndividualBeneficiary.title = Are you sure you want to remove {0}?
 removeIndividualBeneficiary.heading = Are you sure you want to remove {0}?
 removeIndividualBeneficiary.error.required = Select yes if you want to remove the individual beneficiary
+
+charityOrTrust.title = Is the beneficiary a charity or trust?
+charityOrTrust.heading = Is the beneficiary a charity or trust?
+charityOrTrust.checkYourAnswersLabel = Is the beneficiary a charity or trust?
+charityOrTrust.hint = As named in the trust deed
+charityOrTrust.error.required = Select the type of beneficiary you need to add
+
+charityOrTrustTypeOfBeneficiary.charity = Charity
+charityOrTrustTypeOfBeneficiary.trust = Trust
+
+charityBeneficiary.name.title = What is the charity’s name?
+charityBeneficiary.name.heading = What is the charity’s name?
+charityBeneficiary.name.checkYourAnswersLabel = What is the charity’s name?
+charityBeneficiary.name.error.required = Enter the charity’s name
+charityBeneficiary.name.error.invalidFormat = Charity’s name must only include letters a to z, numbers, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
+charityBeneficiary.name.error.length = Charity’s name must be 56 characters or less
+
+charityBeneficiary.discretionYesNo.title = Do the trustees have discretion over how much income {0} may get?
```

Comment:
All the `.title` should not have `{0}` otherwise Personal Identifiable Information (PII) will go into GA.